StudyPactOnboardingViewController.swift

```diff
@@ -0,0 +1,167 @@
+        let numberImage = UIImageView()
+        let descriptionText = UILabel()
+        self.view.addSubview(signUpLabel)
+        self.view.addSubview(bearImage)
+        self.view.addSubview(numberImage)
+        self.view.addSubview(descriptionText)
+        
+        
+        
+        bearImage.image = UIImage(named: "Onboarding1")
+        bearImage.frame = CGRect(x: 0, y: 0, width: 309, height: 321)
+        bearImage.translatesAutoresizingMaskIntoConstraints = false
+        bearImage.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        bearImage.centerYAnchor.constraint(equalTo: view.centerYAnchor, constant: -100).isActive = true
+        
+        
+        signUpLabel.text = "Sign Up"
+        signUpLabel.font = UIFont(name: "Raleway-Bold", size: 40)
+        signUpLabel.translatesAutoresizingMaskIntoConstraints = false
+        signUpLabel.centerXAnchor.constraint(equalTo: self.view.centerXAnchor, constant: 20).isActive = true
+        signUpLabel.centerYAnchor.constraint(equalTo: self.view.centerYAnchor, constant: 65).isActive = true
+        
+        numberImage.image = UIImage(named: "Onboarding1Num")
+        numberImage.frame = CGRect(x: 0, y: 0, width: 34, height: 29)
+        numberImage.translatesAutoresizingMaskIntoConstraints = false
+        numberImage.rightAnchor.constraint(equalTo: signUpLabel.leftAnchor, constant: -20).isActive = true
+        numberImage.centerYAnchor.constraint(equalTo: signUpLabel.centerYAnchor).isActive = true
+        
+        descriptionText.text = "Complete your profile in less than 5 minutes to get the best pairing"
+        descriptionText.font = UIFont(name: "Raleway-Regular", size: 18)
+        descriptionText.frame = CGRect(x: 0, y: 0, width: 300, height: 50)
+        descriptionText.numberOfLines = 0
+        descriptionText.textColor = .gray
+        descriptionText.textAlignment = .center
+        descriptionText.translatesAutoresizingMaskIntoConstraints = false
+        descriptionText.centerXAnchor.constraint(equalTo: self.view.centerXAnchor).isActive = true
+        descriptionText.topAnchor.constraint(equalTo: signUpLabel.bottomAnchor, constant: 30).isActive = true
+        descriptionText.widthAnchor.constraint(equalToConstant: 300).isActive = true
+
+        
+    }
+
+}
+
+class StudyPactOnboarding2ViewController: UIViewController {
+
+    let pageNumber = 1
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        let signUpLabel = UILabel()
+        let bearImage = UIImageView()
+        let numberImage = UIImageView()
+        let descriptionText = UILabel()
+        self.view.addSubview(signUpLabel)
+        self.view.addSubview(bearImage)
+        self.view.addSubview(numberImage)
+        self.view.addSubview(descriptionText)
+        
+        
+        bearImage.image = UIImage(named: "Onboarding2")
+        bearImage.frame = CGRect(x: 0, y: 0, width: 309, height: 321)
```

Comment:
Just commenting on one image - Probably better to constrain the image + label + description in a stackview or smt since the items don't dynamically adjust to screen size right now

Reply:
Do you mean place the items inside of a stack view, then constrain the stack view in the view, and then constrain the individual things inside the stack view?